src/utils/request.ts

```diff
@@ -46,10 +49,15 @@ export function parseConditionalHeaders(headers: Headers): ConditionalHeaders {
     ? headers.get('if-none-match')!.replaceAll('"', '')
     : undefined;
 
-  const ifUnmodifiedSince = headers.has('if-unmodified-since')
+  let ifUnmodifiedSince = headers.has('if-unmodified-since')
     ? new Date(headers.get('if-unmodified-since')!)
     : undefined;
 
+  if (!(ifUnmodifiedSince instanceof Date)) {
+    // Invalid date
```

Comment:
```suggestion
```
Nit: with the suggested method call being `isValidDate` I imagine this comment is redundant now